### ISSUE TITLE
nfs server on the bastion 

### ISSUE THREAD

**Issue by dwojciec** [RedHat-EMEA-SSA-Team/stc#52]
possibility to add a nfs server on the bastion if we can't use OCS due to the number of disks attached to each server.

**Comment by blues-man** [comment]
Makes sense, but I would put instructions on how to do it in the "Hacking" section thus one manipulating env.yml can use NFS, but by default we keep pushing OCS
